src/js/views/patients/patient/sidebar/patient-sidebar.scss

```diff
@@ -1,15 +1,27 @@
 .patient-sidebar {
-  padding: 40px 24px 0;
+  border-left: 1px solid #e6e6e6;
+  padding: 54px 24px 0;
   position: relative;
 }
 
 .patient-sidebar__name {
+  border-bottom: 1px solid $black-90;
   font-size: 24px;
   line-height: 1em;
+  padding-bottom: 16px;
+}
+
+.patient-sidebar__icon {
+  position: absolute;
+  top: 16px;
+
+  .icon {
+    font-size: 20px;
+  }
 }
 
 .patient-sidebar__menu {
   position: absolute;
   right: 24px;
-  top: 8px;
+  top: 16px;
```

Comment:
Adds more padding space above the icon/ellipsis, makes them align with the context trail text, and matches the amount of padding we use for the worklist patient sidebar.